lib/Extractor/Candidates.cpp

```diff
@@ -847,6 +871,64 @@ void ExtractExprCandidates(Function &F, const LoopInfo *LI, DemandedBits *DB,
   for (auto &BB : F) {
     std::unique_ptr<BlockCandidateSet> BCS(new BlockCandidateSet);
     for (auto &I : BB) {
+      if (PrintNegAtReturn && isa<ReturnInst>(I)) {
+        auto V = I.getOperand(0);
+        auto DL = F.getParent()->getDataLayout();
+        bool Negative = 0;
+        Negative = isKnownNegative(V, DL);
+        if (Negative)
+          llvm::outs() << "known at return: " << "(negative)" << "\n";
+        else
+          llvm::outs() << "known at return: " << "" << "\n";
```

Comment:
how about just print something like: "known negative?" followed by true or false? otherwise all these blank "known at return:" are hard to interpret